Tone/instrument/CymbalSynth.js

```diff
@@ -0,0 +1,304 @@
+
+define(["Tone/core/Tone", "Tone/source/Oscillator", "Tone/instrument/Instrument", 
+	"Tone/component/AmplitudeEnvelope", "Tone/component/Filter", 
+	"Tone/signal/Signal", "Tone/signal/Multiply", "Tone/signal/Scale",
+	"Tone/source/PWMOscillator", "Tone/instrument/SimpleFM"],
+function(Tone){
+
+	"use strict";
+	/**
+	 *  @class  CymbalSynth is composed of 6 Tone.FMSynths, with their 
+	 *  carrier set to a square wave, and the modulator set to a pulse 
+	 *  wave. 
+	 *  Inspiration from [Sound on Sound](http://www.soundonsound.com/sos/jul02/articles/synthsecrets0702.asp).
+	 *
+	 *  @constructor
+	 *  @extends {Tone.Instrument}
+	 *  @param {options} [options] the options availble for the synth
+	 *                             see defaults below
+	 *  @example
+	 * var cymbal = new Tone.CymbalSynth().toMaster();
+	 * cymbal.harmonicity.value = 10;
+	 * cymbal.triggerAttack();
+	 */
+	Tone.CymbalSynth = function(options){
+
+		options = this.defaultArg(options, Tone.CymbalSynth.defaults);
+		Tone.Instrument.call(this, options);
+
+		/**
+		 * The initial strike component of the cymbal hit.
+		 * @type {CymbalComponent}
+		 */
+		this.strike = new CymbalComponent({
+			"volume" : options.strike.volume,
+			"cutoff" : options.strike.cutoff,
+			"frequencyScalar" : 1.886,
+			"resonance" : options.strike.resonance,
+			"resonanceScalar" : 0.533,
+			"envelope" : {
+				"attack": options.strike.envelope.attack, 
+				"decay": options.strike.envelope.decay, 
+			}
+		}).connect(this.output);
+
+		/**
+		 * The body portion of the cymbal hit.
+		 * @type {CymbalComponent}
+		 */
+		this.body = new CymbalComponent({
+			"volume" : options.body.volume,
+			"cutoff" : options.body.cutoff,
+			"frequencyScalar" : 1.643,
+			"resonance" : options.body.resonance,
+			"resonanceScalar" : 0.333,
+			"envelope" : {
+				"attack": options.body.envelope.attack, 
+				"decay": options.body.envelope.decay, 
+			}
+		}).connect(this.output);
+
+		/**
+		 *  [harmonicity description]
+		 *  @type {Positive}
+		 *  @signal
+		 */
+		this.harmonicity = new Tone.Signal(options.harmonicity);
+		this.harmonicity.units = Tone.Type.Positive;
```

Comment:
you could assign the units in the constructor: 

`new Tone.Signal(options.harmonicity, Tone.Type.Positive)`

I think the type is actually `NormalRange` since it is connected to a `Tone.Scale` which only takes values between 0-1.